x-pack/test_serverless/functional/test_suites/search/common_configs/config.group3.ts

```diff
@@ -12,9 +12,9 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
 
   return {
     ...baseTestConfig.getAll(),
-    testFiles: [require.resolve('../../common/visualizations/group2')],
+    testFiles: [require.resolve('../../common/visualizations/group3')],
```

Comment:
The different layers of grouping are admittedly a bit confusing - but this change needs to be reverted.
The number of the project specific common config (in this case `config.group3.ts`) does not necessarily match with the arbitrary numbers that are used to group common tests into smaller chucks (in this case `common/visualizations/group2`).

Reply:
I think I got it right [now](https://github.com/elastic/kibana/pull/180127/commits/af5ffa3897c2eee5dbfcfa2ce19cd4f1e0c4cf52) boss.